### PR TITLE
add helper for removing module names section

### PR DESCRIPTION
**Pull request by jwasinger**
Let me know if this would be accepted and I will add tests.

### CODE REVIEW

**Comment by pepyakin** [comment]
Hey, thanks for the PR!

The code makes sense although it seems to me a bit niche and implementable without this function. So this makes me wonder what is your use-case?

**Comment by jwasinger** [comment]
https://github.com/wasmx/wasm-chisel has a wasm sanitization tool which strips the name section from the module.  I understand if this is too niche to merge into the codebase.

File: src/elements/module.rs [hunk 1]
```diff
@@ -272,6 +272,21 @@ impl Module {
 		}
 	}
 
+	/// Removes the module's names section.
+	pub fn clear_names_section(&mut self) {
+		let sections = self.sections_mut();
+		let mut rmidx = sections.len();
+		for (index, section) in sections.iter_mut().enumerate() {
+			if let Section::Name(_) = section {
```
Comment (axic) [inline src/elements/module.rs]: This should also delete unparsed names section (`CustomSection("name")`).

**Comment by axic** [comment]
> The code makes sense although it seems to me a bit niche and the functionality seems to be implementable without this function. 

I agree it is niche, but `clear_start_section` and `clear_custom_section` also exists, which may be considered similarly niche.

One reason to consider this PR is that the names section is a bit special, because it has a different section type once parsed.

**Pull request closed without merging**